feature-libs/cart/import-export/components/import-to-cart/import-entries-dialog/import-entries-dialog.component.html

```diff
@@ -27,10 +27,27 @@
       </div>
 
       <!-- Modal Body -->
-      <cx-import-entries-form
+
+      <ng-container
         *ngIf="formState; else importSummary"
-        (submitEvent)="importProducts($event)"
-      ></cx-import-entries-form>
+        [ngSwitch]="placement$ | async"
+      >
+        <ng-container *ngSwitchCase="Placement.SAVED_CARTS">
```

Comment:
Do we need to wrap those components into `ng-container`?

I think following looks cleaner:
```
      <ng-container
        *ngIf="formState; else importSummary"
        [ngSwitch]="placement$ | async"
      >
        <cx-import-to-saved-cart-form
          (submitEvent)="importProducts($event)"
          *ngSwitchCase="Placement.SAVED_CARTS"
        ></cx-import-to-saved-cart-form>

        <cx-import-entries-form
          (submitEvent)="importProducts($event)"
          *ngSwitchCase="Placement.CART"
        ></cx-import-entries-form>

        <cx-import-entries-form
          (submitEvent)="importProducts($event)"
          *ngSwitchDefault
        ></cx-import-entries-form>
      </ng-container>
```

Reply:
good catch, removed all unused `ng-container`s